frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -135,7 +135,7 @@ action.GitMachete.BaseSlideInBelowAction.dialog.slide-in.ok-button=Slide In
 action.GitMachete.BaseSlideInBelowAction.dialog.slide-in.title=Slide In Branch Below
 action.GitMachete.BaseSlideInBelowAction.dialog.slide-in.no-branch-selected=A branch should be selected.
 action.GitMachete.BaseSlideInBelowAction.dialog.slide-in.error.slide-in-under-itself=Cannot slide in the branch under itself.
-action.GitMachete.BaseSlideInBelowAction.dialog.slide-in.placeholder=Specify branch
+action.GitMachete.BaseSlideInBelowAction.dialog.slide-in.placeholder=Specify a new or an existing branch
```

Comment:
Here's how it looks like now...

<img width="479" alt="image" src="https://user-images.githubusercontent.com/3383210/212372219-5fa0033a-c8c3-4928-be56-94d8764f1dff.png">

What else can be changed... `Slide In New or Existing Branch` in the dialog title? although that would be probably too much

Reply:
I think it's now more precise and looks much better - should be enough. 

However, if we want to verify whether the changed title is better you could provide a screenshot with the changed version.

Reply:
<img width="541" alt="image" src="https://user-images.githubusercontent.com/3383210/212376951-b99362bd-1416-4d64-b20e-5bcacf188766.png">

WDYT?

Reply:
+1 to the changed title